wordpress-mobile/WordPress-Android
# Add random value to the MySiteImprovementsFeatureConfig.kt based on user ID

This is a simple solution that fixes the unreliability of the dynamic cards experiment we're experiencing now. It uses simple function to extract the value from the user ID.

To test:
- sign up with a user
- Notice the my site flag is either On or Off (based on User ID) and the dynamic cards or quick start is shown
- try with a different user until the value is different
- another option is to try creating sites with your current user

## Regression Notes
1. Potential unintended areas of impact
🤞  none

PR submission checklist:
